homeassistant/components/aemet/sensor.py

```diff
@@ -1,6 +1,4 @@
 """Support for the AEMET OpenData service."""
-from homeassistant.components.sensor import SensorEntity
-
 from .abstract_aemet_sensor import AbstractAemetSensor
```

Comment:
Side note: I suggest moving the abstract sensor entity into the sensor platform. There's no reason to have it in a separate module since no other platform can use it.